### PR TITLE
python2 compatible

### PR DESCRIPTION
**Pull request by dani14-96**
Modified necessary imports and functions to make it compatible and
ready-to-use (not need to modify by user)

### CODE REVIEW

File: pycookiecheat (python2)/pycookiecheat.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+Helpful Links:
+* Chromium Mac os_crypt: http://n8h.me/QWRgK8
+* Chromium Linux os_crypt: http://n8h.me/QWTglz
+* Python Crypto: http://n8h.me/QWTqte
+'''
+
+import sqlite3
+import os.path
+import urlparse #imported urlparse instead of urllib.parse for python2
```
Comment (dani14-96) [inline pycookiecheat (python2)/pycookiecheat.py]: import urllib.parse > import urlparse

File: pycookiecheat (python2)/pycookiecheat.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+        # Encrypted cookies should be prefixed with 'v10' according to the
+        # Chromium code. Strip it off.
+        encrypted_value = encrypted_value[3:]
+
+        # Strip padding by taking off number indicated by padding
+        # eg if last is '\x0e' then ord('\x0e') == 14, so take off 14.
+        # Modified function using ord() for python2.
+        def clean(x):
+            return x[:-ord(x[-1])].decode('utf8')
```
Comment (dani14-96) [inline pycookiecheat (python2)/pycookiecheat.py]: ord() function as specified

File: pycookiecheat (python2)/pycookiecheat.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        )
+    else:
+        raise OSError("This script only works on OSX or Linux.")
+
+    # Generate key from values above
+    key = PBKDF2(my_pass, salt, length, iterations)
+
+    # Part of the domain name that will help the sqlite3 query pick it from the
+    # Chrome cookies
+    domain = urlparse.urlparse(url).netloc #used urlparse instead of urllib.parse for python2
```
Comment (dani14-96) [inline pycookiecheat (python2)/pycookiecheat.py]: urllib.parse.urlparse > urlparse.urlparse

**Comment by n8henrie** [comment]
It looks like these changes would break Python3 support, which is more important to me than Python2 support. I'm fine with supporting Python2 as long as it doesn't get in the way of Python3 -- please modify your commit to support both and submit a new PR.

I would recommend something like

**Comment by dani14-96** [comment]
I have the whole repo modified for python2. You can create a new branch and I PR to that branch, or instead create a new repo. Tell me when you do it and i'll PR

**Comment by n8henrie** [comment]
I made a `py2` branch if you want to make a PR for that branch. Thanks for your work!

**Pull request closed without merging**